euphoria-core/src/main/java/cz/seznam/euphoria/core/client/operator/hint/Util.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2016-2018 Seznam.cz, a.s.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package cz.seznam.euphoria.core.client.operator.hint;
+
+import cz.seznam.euphoria.core.client.dataset.Dataset;
+import cz.seznam.euphoria.core.client.dataset.windowing.MergingWindowing;
+import cz.seznam.euphoria.core.client.operator.Join;
+import cz.seznam.euphoria.core.client.operator.Operator;
+
+import java.util.ArrayList;
+
+public class Util {
```

Comment:
The location of this class seems strange. First of all, it seems that it is executor related, so should be in `executor` package, annotated with `Audience` as appropriate. Moreover, I think that the generic name `Util` could be made more explicit.